langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -263,6 +371,11 @@ public isolated function createText(string data) returns Text = @java:Method {
 
 # Returns a subsequence of an xml value.
 #
+# ```ballerina
+# xml books = xml `<book>HTML</book><book>Ballerina</book><book>Python</book><book>CSS</book>`;
+# books.slice(1, 3) ⇒ <book>Ballerina</book><book>Python</book>
```

Comment:
shall we add an example without end index as well

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39130/commits/3a7088c41159f3c56a37443fa2036905f399eaa5